ic-agent/src/agent/mod.rs

```diff
@@ -258,6 +258,14 @@ pub struct Agent {
     transport: Arc<dyn ReplicaV2Transport>,
 }
 
+impl fmt::Debug for Agent {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
+        f.debug_struct("Agent")
+            .field("ingress_expiry_duration", &self.ingress_expiry_duration)
+            .finish()
```

Comment:
If certain fields are absent, this should be notated in the output via `finish_non_exhaustive`.

Reply:
Good point, thanks!